### PR TITLE
coverd situation when supplier have the same hotel

### PR DESCRIPTION
**Pull request by hasmikharutyunyan**
Logging error, when supplier have the same hotel. This implementation is temporary, so not  guarantee that all that situations are covered. The aim was not throw an exception when on mapping was found that kind of hotels.

### CODE REVIEW

File: HappyTravel.StaticDataMapper.Api/Infrastructure/Logging/LogEvents.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
   {"id": 90302, "name": "PreloadingAccommodationsCancel", "level": "Information", "source": "AccommodationPreloader", "isException": false},
   {"id": 90303, "name": "PreloadingAccommodationsError", "level": "Error", "source": "AccommodationPreloader", "isException": true},
 
-  {"id": 90401, "name": "ConnectorClientError", "level": "Error", "source": "ConnectorClient", "isException": true}
+  {"id": 90400, "name": "ConnectorClientError", "level": "Error", "source": "ConnectorClient", "isException": true},
```
Comment (mingazhev) [inline HappyTravel.StaticDataMapper.Api/Infrastructure/Logging/LogEvents.json:34]: you can use isException = false, in this case you will need to add a string error only

**Review by mingazhev — APPROVED** [review]
with a comment

File: HappyTravel.StaticDataMapper.Api/Infrastructure/Logging/LogEvents.json (hunk 1 above)
Comment (hasmikharutyunyan) [reply HappyTravel.StaticDataMapper.Api/Infrastructure/Logging/LogEvents.json:34]: I done this especially, to differ this logs

Comment (hasmikharutyunyan) [reply HappyTravel.StaticDataMapper.Api/Infrastructure/Logging/LogEvents.json:34]: Sorry, I understand what You mean.

**Pull request merged**